docs/Acceptable-Formulae.md

```diff
@@ -86,6 +86,8 @@ Homebrew formulae should avoid having multiple, separate, upstream projects bund
 
 For more info see [Debian's](https://www.debian.org/doc/debian-policy/ch-source.html#s-embeddedfiles) and [Fedora's](https://docs.fedoraproject.org/en-US/packaging-guidelines/#bundling) stances on this.
 
+Increasingly, though: this can be (too) hard. Homebrew's primary mission is to be useful rather than ideologically pure. If we cannot package something without using vendored upstream versions: so be it; better to have packaged software in Homebrew than not.
```

Comment:
```suggestion
Increasingly, though: this can be (too) hard. Homebrew's primary mission is to be useful rather than ideologically pure. If we cannot package something popular without using vendored upstream versions: so be it; better to have  packaged software that is useful to many in Homebrew than not.
```